include/LightGBM/utils/common.h

```diff
@@ -51,6 +54,13 @@ namespace LightGBM {
 
 namespace Common {
```

Comment:
Since Common is within LightGBM I'll ask to add 2 spaces in all Common namespace. Otherwise extract Common namespace out of LightGBM namespace, for the readability.

Reply:
See https://www.tutorialspoint.com/cplusplus/cpp_namespaces.htm

Reply:
Common exists since ever, I don't want to make a huge diff just to align the namespace, as it would make less evident what really changed in there.